src/create_torrent.cpp

```diff
@@ -332,14 +332,15 @@ namespace libtorrent
 		// a piece_size of 0 means automatic
 		if (piece_size == 0 && !m_merkle_torrent)
 		{
-			const int target_size = 40 * 1024;
-			piece_size = int(fs.total_size() / (target_size / 20));
+			const int hash_size = 20;
+
+			double target_list_size = sqrt(fs.total_size()) * 2;
```

Comment:
this should be ``std::sqrt()``

Reply:
fixed.

Reply:
https://ci.appveyor.com/project/arvidn/libtorrent/build/1.0.8089/job/3r5ac86p4a47qhmw#L238

Reply:
>Would it make sense to just have a table instead?

Indeed, PR updated.
I try keeping `sqrt()` for master branch.